apps/src/music/views/HeaderButtons.tsx

```diff
@@ -86,6 +97,7 @@ const HeaderButtons: React.FunctionComponent<HeaderButtonsProps> = ({
   const currentPackId = useAppSelector(state => state.music.packId);
   const analyticsReporter = useContext(AnalyticsContext);
   const dialogControl = useDialogControl();
+  const {isSignedIn, feedbackLink} = useFeedbackLink();
```

Comment:
The `use` prefix is generally related to hooks in React.  How about `getFeedbackLink`?  

(@sanchitmalhotra126 might have a better idea.)

Reply:
Fair point!  It's hooks all the way down...